packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -1998,7 +1993,12 @@ class WebpackCLI {
             if (!configOptions.plugins) {
                 configOptions.plugins = [];
             }
-
+            const alreadyHasPLugin = configOptions.plugins.find(
+                (plugin) => plugin instanceof CLIPlugin,
+            );
+            if (alreadyHasPLugin) {
+                return configOptions;
+            }
```

Comment:
Rerun `applyOptions` is very bad idea, it has very bad performance and it is not safe too